site-manage/datemanager/tool/src/main/webapp/WEB-INF/templates/index.html

```
@@ -42,6 +42,52 @@ <h1 th:text="#{page.instructions}">Add/Edit Options in batch</h1>
       <a id="importDates" class="btn-link" th:text="#{dateshifter.import.dates.csv}" th:href="@{/date-manager/page/import}">Import Dates</a>
     </div>
 
+    <div class="date-manager-setter mb-3">
+      <div class="sak-banner-error d-none" id="dateBulkSetterError">
+        <span th:text="#{datesetter.error.empty}"></span>
+      </div>
+      <p th:text="#{datesetter.instructions}">Enter one or more dates below, then apply them to all items or expanded sections only. Remember to click Save Changes at the bottom of the page to save your adjustments.</p>
+      <div class="row g-3">
+        <div class="col-sm-6 col-lg-3">
+          <label class="form-label" for="bulk_open_date" th:text="#{column.open.date}">Open Date</label>
+          <input id="bulk_open_date" type="text" class="form-control bulk-date-input"></input>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </input> ]

(tag-pair)